lib/00_core.sh

```diff
@@ -95,8 +95,8 @@ _tranlate_w() {
 
   "yum")
     _TOPT="--downloadonly"
-    if ! rpm -q 'yum-downloadonly' >/dev/null 2>&1; then
-      _error "'yum-downloadonly' package is required when '-w' is used."
+    if ! [[ $(rpm -qa --pipe "grep downloadonly") ]]; then
```

Comment:
I tested with redirect and saw it failed with nothing print, I have no idea now.